PantryServer/Models/Cart.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+
+//This will allow users to place albums in their cart without registering, but they’ll need to register to complete checkout
+namespace PantryServer.Models
+{
+    public class Cart
+    {
+        // Our Cart items will have a string identifier named CartID to allow anonymous shopping
```

Comment:
something smells with this one. you have a product id and a product. why?
also wouldn't a cart have a collection of collections of products. lets work on this on weds.